src/PhpWord/Shared/Html.php

```diff
@@ -472,11 +472,53 @@ protected static function parseCell($node, $element, &$styles)
             $cellStyles['gridSpan'] = $colspan - 0;
         }
 
+        $rowspan = $node->getAttribute('rowspan');
+        if (!empty($rowspan)) {
+            $cellStyles['vMerge'] = 'restart';
+        }
+        $beforespan = $node->getAttribute('beforespan');
```

Comment:
This attribute doesn't exist. Can you use a Real attribute or an another method ?